app/src/helpers/customAreaObjects.js

```diff
@@ -320,6 +320,37 @@ const fetchCustomAreaObjects = async (
     if (typeof mergedConfig[lookup].callbackFunction === 'function') {
       customObjects = mergedConfig[lookup].callbackFunction(mergedData, indicator);
     }
+  } else if (
+    indicator.display.areaIndicator.url === 'https://staging-raster.delta-backend.com/cog/statistics'
```

Comment:
It would be safer to check for mergedConfigs[lookup] everywhere in this function as it also includes inputData configs merged in. The URL check could be generalized a bit, whenever we stop using staging URL from the delta backend.

Reply:
I was thinking what to use for the check and considering if we should add an endpoint identifier and then do a switch on the endpoint.
Not sure i understand what we can use from within the mergedconfig as better identifier

Reply:
mergedConfigs[lookup].areaIndicator.url should be available and then this would cover also the inputData cases (if we have them in the future).

Reply:
Also now just checking if url contains /cog/statistics i guess that should be enough for a check

Reply:
I think it should be `mergedConfigs[lookup].areaIndicator.url` or?